esx_service/vmdk_ops_test.py

```diff
@@ -731,7 +730,7 @@ def setUp(self):
                 self.datastore_path = datastore[2]
                 logging.debug("datastore_name=%s datastore_path=%s", self.datastore_name,
                                                                      self.datastore_path)
-                if len(datastores) >= 1:
+                if len(datastores) >1:
```

Comment:
old code: 
if len(datastores) >=1:
                    datastore1 = datastores[1]    >>>>>> this will cause failure if we have only one datastore in testbed
                    self.datastore1_name = datastore1[0]
                    self.datastoer1_path = datastore[2]
                    logging.debug("Found second datastore: datastore_name=%s datastore_path=%s",
                                  self.datastore1_name, self.datastore1_path)


This new condition enables that code within if block will only be executed if we have more than one datastore.